app/bundles/LeadBundle/Form/Type/ListType.php

```diff
@@ -57,6 +58,12 @@ public function __construct(MauticFactory $factory)
             $this->listChoices[$list['id']] = $list['name'];
         }
 
+        $emails                = $listModel->getUserEmails();
+        $this->emailChoices     = array();
+        foreach ($emails as $email) {
```

Comment:
It can be cast to array by doing `$emails = (array) $listModel->getUserEmails()`, or just performing a check on the emails prior to iterating.